scripts/docs/config.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
same, why not `ts`?

Reply:
Probably it's mine own bias to have JSON config files. Do you think it's better to change it to `config.ts`?

Reply:
Fixed in 85863f0